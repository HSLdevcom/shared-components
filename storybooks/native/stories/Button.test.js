import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  framework: 'react-native',
  storyKindRegex: /^Button$/
});
