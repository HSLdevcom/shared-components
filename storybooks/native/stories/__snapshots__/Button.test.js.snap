// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button default 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    className={undefined}
    large={false}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        },
        Array [
          Object {
            "backgroundColor": "#fff",
            "borderColor": "#b7b7b7",
            "borderRadius": 3.5,
            "borderStyle": "solid",
            "borderWidth": 1,
            "height": 39.375,
            "width": "70%",
          },
          Array [
            Object {},
            Array [
              Object {},
              undefined,
            ],
          ],
        ],
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          Array [
            Object {
              "fontFamily": "Gotham Rounded SSm A, Gotham Rounded SSm B",
              "fontSize": 14,
            },
            Array [
              Object {
                "color": "#007ac9",
                "fontSize": 14,
                "fontWeight": "500",
                "textAlign": "center",
              },
              undefined,
            ],
          ],
        ]
      }
    >
      Default Button
    </Text>
  </View>
</View>
`;

exports[`Storyshots Button primary 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    className={undefined}
    large={false}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        },
        Array [
          Object {
            "backgroundColor": "#007ac9",
            "borderColor": "#007ac9",
            "borderRadius": 35,
            "borderStyle": "solid",
            "borderWidth": 1,
            "height": 43.75,
            "width": "70%",
          },
          Array [
            Object {},
            Array [
              Object {},
              undefined,
            ],
          ],
        ],
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          Array [
            Object {
              "fontFamily": "Gotham Rounded SSm A, Gotham Rounded SSm B",
              "fontSize": 14,
            },
            Array [
              Object {
                "color": "#ffffff",
                "fontSize": 15.75,
                "fontWeight": "500",
                "textAlign": "center",
              },
              undefined,
            ],
          ],
        ]
      }
    >
      Primary Button
    </Text>
  </View>
</View>
`;

exports[`Storyshots Button round button 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    className={undefined}
    large={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        },
        Array [
          Object {
            "backgroundColor": "#fff",
            "borderColor": "#b7b7b7",
            "borderRadius": 35,
            "borderStyle": "solid",
            "borderWidth": 1,
            "height": 39.375,
            "width": "70%",
          },
          Array [
            Object {},
            Array [
              Object {},
              Array [
                Object {
                  "borderColor": "#007ac9",
                  "height": 70,
                  "width": 70,
                },
                undefined,
              ],
            ],
          ],
        ],
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "color": "#007ac9",
            "height": 31.5,
            "justifyContent": "center",
            "width": 31.5,
          },
          undefined,
        ]
      }
    >
      <Svg
        height="36px"
        preserveAspectRatio="xMidYMid meet"
        style={
          Object {
            "fill": "currentColor",
          }
        }
        viewBox="0 0 17 33"
        width="18px"
      >
        <Path
          d="M16.7982286,11.9856261 L11.034563,11.9856261 L11.034563,8.26958518 C11.034563,6.58443921 11.5045179,5.43616736 13.920705,5.43616736 L17,5.43464647 L17,0.229147407 C16.4681955,0.158172546 14.6405928,0 12.5103331,0 C8.06629291,0 5.02603406,2.71276056 5.02603406,7.69367489 L5.02603406,11.9856261 L0,11.9856261 L0,17.8050577 L5.02603406,17.8050577 L5.02603406,32.7381684 L11.034563,32.7381684 L11.034563,17.8050577 L16.047416,17.8050577 L16.7982286,11.9856261 Z"
          fill={undefined}
        />
      </Svg>
    </View>
  </View>
</View>
`;

exports[`Storyshots Button secondary 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    className={undefined}
    large={false}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        },
        Array [
          Object {
            "backgroundColor": "#fff",
            "borderColor": "#848484",
            "borderRadius": 3.5,
            "borderStyle": "solid",
            "borderWidth": 1,
            "height": 39.375,
            "width": "70%",
          },
          Array [
            Object {},
            Array [
              Object {},
              undefined,
            ],
          ],
        ],
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          Array [
            Object {
              "fontFamily": "Gotham Rounded SSm A, Gotham Rounded SSm B",
              "fontSize": 14,
            },
            Array [
              Object {
                "color": "#848484",
                "fontSize": 14,
                "fontWeight": "500",
                "textAlign": "center",
              },
              undefined,
            ],
          ],
        ]
      }
    >
      Secondary Button
    </Text>
  </View>
</View>
`;
